airflow/ui/src/views/Docs.tsx

```diff
@@ -18,12 +18,171 @@
+    { path: 'oracle', name: 'Oracle' },
+    { path: 'pagerduty', name: 'Pagerduty' },
+    { path: 'papermill', name: 'Papermill' },
+    { path: 'plexus', name: 'Plexus' },
+    { path: 'postgres', name: 'PostgreSQL' },
+    { path: 'presto', name: 'Presto' },
+    { path: 'qubole', name: 'Qubole' },
+    { path: 'redis', name: 'Redis' },
+    { path: 'salesforce', name: 'Salesforce' },
+    { path: 'samba', name: 'Samba' },
+    { path: 'segment', name: 'Segment' },
+    { path: 'sendgrid', name: 'Sendgrid' },
+    { path: 'sftp', name: 'SFTP' },
+    { path: 'singularity', name: 'Singularity' },
+    { path: 'slack', name: 'Slack' },
+    { path: 'snowflake', name: 'Snowflake' },
+    { path: 'sqlite', name: 'SQLite' },
+    { path: 'ssh', name: 'SSH' },
+    { path: 'telegram', name: 'Telegram' },
+    { path: 'vertica', name: 'Vertica' },
+    { path: 'yandex', name: 'Yandex' },
+    { path: 'zendesk', name: 'Zendesk' },
+  ];
+
+  return (
+    <AppContainer>
+      <Box mx="auto" my={8} maxWidth="900px">
+        <Flex mt={8}>
+          <Box flex="1">
+            <Heading as="h1">Documentation</Heading>
+            <Text mt={4}>
+              Apache Airflow Core, which includes webserver, scheduler, CLI and other components
+              that are needed for minimal Airflow&nbsp;installation.
+            </Text>
+            <Button
+              as="a"
+              href="https://airflow.apache.org/docs/apache-airflow/stable/index.html"
+              variant="solid"
+              rightIcon={<FiExternalLink />}
+              mt={4}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Apache Airflow Docs
+            </Button>
+          </Box>
+          <Box ml={8} p={4} bg={useColorModeValue('gray.100', 'gray.700')} borderRadius="md">
+            <Heading as="h3" size="sm">Links</Heading>
+            <List mt={4} spacing={2}>
+              <ListItem>
+                <Link href="https://airflow.apache.org/" isExternal color="teal.500">
+                  <Icon as={FiGlobe} mr={1} />
+                  Apache Airflow Website
+                </Link>
+              </ListItem>
+              <ListItem>
+                <Link href="https://github.com/apache/airflow" isExternal color="teal.500">
+                  <Icon as={FaGithub} mr={1} />
+                  apache/airflow on GitHub
+                </Link>
+              </ListItem>
+            </List>
+          </Box>
+        </Flex>
+        <Box mt={10}>
+          <Heading as="h3" size="lg">REST API Reference</Heading>
+          <Flex mt={4}>
+            <Button
+              as="a"
+              href="http://127.0.0.1:28080/api/v1/ui/"
+              variant="outline"
+              rightIcon={<FiExternalLink />}
+              mr={2}
+            >
+              Swagger
+            </Button>
+            <Button
+              as="a"
+              href="http://127.0.0.1:28080/redoc"
```

Comment:
We can use `process.env.API_URl` here.

Reply:
We'll have to add a new key as that one includes `/api/v1/`

Reply:
Ok. so lets make a new env var for it.